src/fastaRecs.py

```diff
@@ -0,0 +1,77 @@
+import argparse
+from Genome import Genome
+from utils import *
+
+# This method provides a way to get the proccessed content of a genome
+def getGenomes(c):
+    genomes = []
+
+    length = len(c)
+    i = 0
+
+    # Find the symbol that tells us the beginning of a chromosome
+    while  c[i] != '>':
+        i += 1
+        if i >= length:
+            return
+    
+    name = None
+    genomeChain = None
+    chain = ""
+
+    while(i<length):        
+        i += 1
+
+        # Find the beginning of the name
+        while isSpaceType(c[i]):
+            i += 1
+            if i >= length:
+                return genomes
+        
+        # Store all the name
+        while c[i] != '\n':
+            chain += c[i]
+            i += 1
+            if i >= length:
+                break
```

Comment:
Here, you could just as well just find the next `\n` after the start of the name, i.e., where `i` is at the beginning of the `while i < length:` loop. You can strip the result after (with `res.strip()`) so you don't have to explicitly skip past whitespace, which in any case will be slower because you are writing your loop in Python while `strip()` is written in C.

If you are currently at position `i` in a string `x` and you want to find the next occurrence of `"foo"` you can again use `find`: `x.find("foo", start = i)`.